internal/http.go

```diff
@@ -8,19 +8,11 @@ package internal // import "go.mongodb.org/mongo-driver/internal"
 
 import (
 	"net/http"
-	"time"
 )
 
 // DefaultHTTPClient is the default HTTP client used across the driver.
 var DefaultHTTPClient = &http.Client{
-	// TODO(GODRIVER-2623): Use "http.DefaultTransport.Clone" once we change the minimum supported Go version to 1.13.
-	Transport: &http.Transport{
-		Proxy:                 http.ProxyFromEnvironment,
-		MaxIdleConns:          100,
-		IdleConnTimeout:       90 * time.Second,
-		TLSHandshakeTimeout:   10 * time.Second,
-		ExpectContinueTimeout: 1 * time.Second,
-	},
+	Transport: http.DefaultTransport.(*http.Transport).Clone(),
```

Comment:
We need to omit this change until after the v1.11.0 release. We're changing the documented minimum Go version in v1.11.0, but not including any actual breaking changes until the next release. Please remove this and add it in a separate PR which we can merge after the v1.11.0 release (scheduled for today).

Reply:
ah got it